helm/a3s/templates/service.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: a3s
+  labels:
+    app: a3s
+    type: core
+spec:
+  selector:
+    app: a3s
+  clusterIP: None
```

Comment:
should be configured from values
we also need to allow to configure the exposed port. as well as load balancer mode